versions.yml

```diff
@@ -1,4 +1,5 @@
 versions:
+  25: "PostgreSQL 9.6.6"
```

Comment:
Please, update the versions.yml to have:
old: 24
older: 18
Also remove the version 25. It will be automatically added by the pipeline job that creates the final release.

Reply:
Done!